e2e/libs/storageclass/storageclass.py

```diff
@@ -50,3 +51,28 @@ def cleanup(self):
         storage_classes = self.api.list_storage_class(label_selector="test.longhorn.io=e2e")
         for item in storage_classes.items:
             self.delete(item.metadata.name)
+
+    def set_storageclass_default_state(self, name, make_default=True):
+        if make_default:
+            logging(f"Set default annotation for storageclass {name}")
+        else:
+            logging(f"Remove default annotation for storageclass {name}")
+
+        value = '"true"' if make_default else 'null'
+        patch_cmd = [
+            "kubectl", "patch", "storageclass", name,
+            "-p", f'{{"metadata": {{"annotations": {{"storageclass.kubernetes.io/is-default-class": {value}}}}}}}'
```

Comment:
Is it possible to use python k8s api [patch_storage_class](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/StorageV1Api.md#patch_storage_class)? It allows us to get rid of tedious `}}}}}}}`.

Reply:
Updated!